src/System.Runtime.Numerics/tests/Complex/ComplexTestSupport.cs

```diff
@@ -11,14 +10,10 @@ namespace ComplexTestSupport
     public static class Support
     {
         private static Random s_random;
-        public static int randomSample;
```

Comment:
@ellismg made the opposite feedback earlier and I agree with him - Support.RandomSampleCount doesn't mean '3' to me, and it was only used in a half-dozen places, so better to just inline it where it's used.